client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -266,106 +269,105 @@ const PlanComparisonGridHeader: React.FC< PlanComparisonGridHeaderProps > = ( {
 				className="plan-comparison-grid__header plan-comparison-grid__interval-toggle"
 				isInSignup={ isInSignup }
 			/>
-			{ visiblePlansProperties.map(
```

Comment:
When reviewing this file on GH, turn on the "Hide whitespace" options since this is mostly a auto-linted change.
<img width="753" alt="image" src="https://user-images.githubusercontent.com/5436027/216951468-de0c3b7c-1bff-424c-b013-896b3399b95e.png">